composer.json

```
@@ -22,6 +22,6 @@
         }
   ],
   "require": {
-    "php": "^8.0"
+    "php": "^8.1"
```

Add drupal 10 requirement too
```suggestion
    "php": "^8.1",
    "drupal/core": "^10.1"
```